crate/server/src/config/command_line/bootstrap_server_config.rs

```diff
@@ -10,14 +10,22 @@ pub struct BootstrapServerConfig {
     #[clap(long, env("KMS_USE_BOOTSTRAP_SERVER"), default_value("false"))]
     pub use_bootstrap_server: bool,
 
-    /// The name that will be the CN
-    /// in the bootstrap server self-signed certificate
+    /// Subject as an RFC 4514 string for the RA-TLS certificate
+    /// in the bootstrap server
     #[clap(
         long,
-        env("KMS_BOOTSTRAP_SERVER_COMMON_NAME"),
-        default_value("cosmian.kms")
+        env("KMS_BOOTSTRAP_SERVER_SUBJECT"),
+        default_value("CN=cosmian.kms,O=Cosmian Tech,C=FR,L=Paris,ST=Ile-de-France")
```

Comment:
for my knowledge and just to make sure, this CN is only used temporary for the bootstrap process?

Reply:
Indeed